Apsitvarkom.Api/Controllers/PollutedLocationController.cs

```diff
@@ -154,30 +154,32 @@ public async Task<ActionResult<PollutedLocationResponse>> Create(PollutedLocatio
         return CreatedAtAction(nameof(GetById), new { response.Id }, response);
     }
 
-    [ProducesResponseType(StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status204NoContent)]
     [ProducesResponseType(StatusCodes.Status400BadRequest)]
     [ProducesResponseType(StatusCodes.Status404NotFound)]
     [ProducesResponseType(StatusCodes.Status500InternalServerError)]
     [HttpDelete("Delete")]
-    public async Task<ActionResult<PollutedLocationResponse>> Delete([FromQuery] ObjectIdentifyRequest pollutedLocationIdentifyRequest)
+    public async Task<ActionResult> Delete([FromQuery] ObjectIdentifyRequest pollutedLocationIdentifyRequest)
     {
         var validationResult = await _objectIdentifyValidator.ValidateAsync(pollutedLocationIdentifyRequest);
         if (!validationResult.IsValid) return BadRequest(validationResult.Errors.Select(e => e.ErrorMessage).ToList());
 
-        PollutedLocation? location;
+        bool locationExists;
         try
         {
-            location = await _repository.GetByPropertyAsync(x => x.Id == pollutedLocationIdentifyRequest.Id);
+            locationExists = await _repository.ExistsByPropertyAsync(x => x.Id == pollutedLocationIdentifyRequest.Id);
         }
         catch (Exception)
         {
             return StatusCode(StatusCodes.Status500InternalServerError);
         }
         
-        if (location is null) return NotFound($"Polluted location with the specified id '{pollutedLocationIdentifyRequest.Id}' was not found.");
-
-        var mappedLocation = _mapper.Map<PollutedLocationResponse>(location);
-        if (mappedLocation is null) return StatusCode(StatusCodes.Status500InternalServerError);
+        if (locationExists is false) return NotFound($"Polluted location with the specified id '{pollutedLocationIdentifyRequest.Id}' was not found.");
+        
+        var location = new PollutedLocation
+        {
+            Id = (Guid)pollutedLocationIdentifyRequest.Id!
+        };
```

Comment:
It would be worthwhile to change this to an automapper mapping?

Reply:
I think you can just map the `Id` and then do `ForAllOtherMembers(opt => opt.Ignore());`

Reply:
`ForAllOtherMembers` is deprecated as of AutoMapper 11 - used what was suggested in the migration guide:
![image](https://user-images.githubusercontent.com/86420873/204359295-67e34bfe-6b90-4ea1-80d4-5f1981e35f23.png)